install.sh

```diff
@@ -94,11 +94,30 @@ nvm_download() {
 install_nvm_from_git() {
   local INSTALL_DIR
   INSTALL_DIR="$(nvm_install_dir)"
+  local NVM_VERSION
+  NVM_VERSION="${NVM_INSTALL_VERSION:-$(nvm_latest_version)}"
+  local FETCH_VERSION
+  FETCH_VERSION="refs/tags/$NVM_VERSION"
+  if [ -n "${NVM_INSTALL_VERSION:-}" ]; then
+    # Check if version is a tag
+    if [ "$(command git ls-remote "$(nvm_source "git")" "refs/tags/$NVM_VERSION" | wc -l)" != "0" ]; then
+      :
+    # Check if version is a ref
+    elif [ "$(command git ls-remote "$(nvm_source "git")" "$NVM_VERSION" | wc -l)" != "0" ]; then
```

Comment:
why not `command git ls-remote "$(nvm_source "git")" "${NVM_INSTALL_VERSION}" | grep`? that would tell you if it was `refs/tags/` or `refs/heads/.

Reply:
I dont like grep so much here, as you can the in my other PR failing, just adding nvm_grep make the CI failing...
Also, on what would I grep ? grep "$NVM_VERSION" ? I am not sure about the difference with using wc -l then ?

Reply:
Ah sorry yes, did not get your comment well in fact. Yes in fact I could put them both together as there is no difference. It was just based on my other PR where I was doing a difference between a tag and a ref.

Reply:
It seems important for us to know the difference, but it seems like a single ls-remote call would provide it if we used grep or similar to filter the results.